mapswipe_workers/tests/unittests/test_firebase_connection.py

```diff
@@ -0,0 +1,24 @@
+import unittest
+
+from mapswipe_workers import auth
+
+
+class TestFirebaseConnection(unittest.TestCase):
+    def setUp(self):
+        self.fb_db = auth.firebaseDB()
+
+    def tearDown(self):
+        self.fb_db = None
+
+    def test_create_project(self):
+        ref = self.fb_db.reference("/v2/test")
+        ref.push("test")
```

Comment:
I think if you push something here, you should also remove it during tear down?

maybe there is a simpler test to check of the Firebase Connection is working.

Reply:
Yes there is a simpler test indeed. Now the tests just gets the keys at root with .get(shallow=True).